aws-mssql.yml

```
@@ -0,0 +1,96 @@
+---
+version: 1
+name: csb-aws-mssql
+id: 8b17758e-37a9-4c1c-af84-971d4a5552c1
+description: CSB Amazon RDS for MSSQL
+display_name: CSB Amazon RDS for MSSQL
+image_url: file://service-images/csb.png
+documentation_url: https://docs.vmware.com/en/Cloud-Service-Broker-for-VMware-Tanzu/index.html
+provider_display_name: VMware
+support_url: https://aws.amazon.com/sql/
+tags: [aws, mssql, beta]
+plan_updateable: true
+provision:
+  plan_inputs: [] # Expected: mssql_version, storage_gb
+  user_inputs:
+  - field_name: db_name
+    type: string
+    details: Name for your database
+    default: vsbdb
+    constraints:
+      maxLength: 64
+    prohibit_update: true
+  - field_name: instance_name
+    type: string
+    details: Name for your MSSQL instance
+    default: csb-mssql-${request.instance_id}
+    constraints:
+      maxLength: 98
+      minLength: 6
+      pattern: ^[a-z][a-z0-9-]+$
+    prohibit_update: true
+  - field_name: region
+    type: string
+    details: The region of AWS.
+    default: us-west-2
+    constraints:
+      examples:
+      - us-central1
+      - asia-northeast1
+      pattern: ^[a-z][a-z0-9-]+$
+    prohibit_update: true
+  - field_name: aws_access_key_id
```

```suggestion
  - field_name: aws_access_key_id
    default: ${config("aws.access_key_id")}
```

In other services this property uses this default value.